policyengine_us/tests/policy/baseline/gov/states/sc/tax/income/credits/sc_cdcc.yaml

```diff
@@ -9,7 +9,7 @@
     # 1,000*0.07 = 70
 
 - name: One cdcc eligible person, with $4000 spending on federal child care expense
```

Comment:
could you also add a unit test for 2022 exceeding the cap? to ensure the offset works as intended